Library/Dungeon Fantasy RPG/Dungeon Fantasy RPG Equipment Modifiers.eqm

```diff
@@ -0,0 +1,177 @@
+					"based_on_id": "42a9c6fa-0243-4c1d-95c6-7c2f460bedc9",
+					"based_on_hash": "c5Ejkdc8F0umXT3v8diMtOVlivkGHadiF7WZsp+v40A",
+					"name": "Dragonhide (+2 DR)",
+					"reference": "DFA110",
+					"cost_type": "to_base_cost",
+					"cost": "+35 CF",
+					"weight_type": "to_base_weight",
+					"weight": "x1.5",
+					"notes": "Heavy leather; additional +2 DR vs fire"
+				},
+				{
+					"type": "eqp_modifier",
+					"version": 1,
+					"id": "6e1f8944-5766-44cc-a84f-80a8438ae3c5",
+					"based_on_id": "964a3837-ea28-4c2d-b94b-7377042fcd8c",
+					"based_on_hash": "wZC34QVqQdt0/ERICSrS4FWZZ09puWvAkzEezCriwUs",
+					"name": "Dragonhide (+3 DR)",
+					"reference": "DFA110",
+					"cost_type": "to_base_cost",
+					"cost": "+41 CF",
+					"weight_type": "to_base_weight",
+					"weight": "x1.75",
+					"notes": "Heavy leather; additional +3 DR vs fire"
+				},
+				{
+					"type": "eqp_modifier",
+					"version": 1,
+					"id": "d0c640a4-b4ee-47fa-9099-9e0a6456996c",
+					"based_on_id": "6e1f8944-5766-44cc-a84f-80a8438ae3c5",
+					"based_on_hash": "37r8IxhlaPfywO0KfMzjfZH8QqmltsedIJqoQjwxDSI",
+					"name": "Dragonhide (+4 DR)",
+					"reference": "DFA110",
+					"cost_type": "to_base_cost",
+					"cost": "+47 CF",
+					"weight_type": "to_base_weight",
+					"weight": "x2",
+					"notes": "Heavy leather; additional +4 DR vs fire"
+				},
+				{
+					"type": "eqp_modifier",
+					"version": 1,
+					"id": "53b65396-65a2-410e-82ae-25c69ecdd641",
+					"name": "Dwarven (+1 DR)",
+					"reference": "DFA110",
+					"cost_type": "to_base_cost",
+					"cost": "+4 CF",
+					"notes": "Plate"
+				},
+				{
+					"type": "eqp_modifier",
+					"version": 1,
+					"id": "8fac7fc7-3c6a-4c11-b1de-0586d56f1f04",
+					"name": "Fine",
+					"reference": "DFA110",
+					"cost_type": "to_base_cost",
+					"cost": "+9 CF",
+					"weight_type": "to_base_weight",
+					"weight": "x3/4"
+				},
+				{
+					"type": "eqp_modifier",
+					"version": 1,
+					"id": "d4f0ad21-22a9-4a81-8954-329c2e27bdda",
+					"name": "Giant-Spider Silk (+1 DR)",
+					"reference": "DFA110",
+					"cost_type": "to_base_cost",
+					"cost": "+99 CF",
+					"weight_type": "to_base_weight",
+					"weight": "+0 lb",
+					"notes": "Light cloth"
+				},
+				{
+					"type": "eqp_modifier",
+					"version": 1,
+					"id": "903fa020-4a41-4bdf-9785-664685343be7",
+					"name": "Meteoric",
+					"reference": "DFA110",
+					"cost_type": "to_base_cost",
+					"cost": "+19 CF",
+					"notes": "Any metal"
+				},
+				{
+					"type": "eqp_modifier",
+					"version": 1,
+					"id": "1a955050-1485-4a3e-98b9-1d28444c9b08",
+					"name": "Orichalcum",
+					"reference": "DFA110",
+					"cost_type": "to_base_cost",
+					"cost": "+29 CF",
+					"weight_type": "to_base_weight",
+					"weight": "x1/3",
+					"notes": "Any plate"
+				},
+				{
+					"type": "eqp_modifier",
+					"version": 1,
+					"id": "da7aab67-2adb-455c-8371-2ac173df1f2c",
+					"name": "Ornate +1",
```

Comment:
I've not read up on this, but doesn't this grant a reaction modifier of some sort? If so, a suitable feature granting that modifier should be added to this equipment modifier (and the other Ornates)